slsa/provenance.go

```diff
@@ -58,7 +64,8 @@ func HostedActionsProvenance(w WorkflowRun) (*intoto.ProvenanceStatement, error)
 			Materials:   w.Materials,
 			// TODO(https://github.com/slsa-framework/slsa-github-generator/issues/8): support more metadata fields.
 			Metadata: &slsa.ProvenanceMetadata{
-				Completeness: w.Completeness,
+				BuildInvocationID: buildInvocationID,
+				Completeness:      w.Completeness,
```

Comment:
i'm just curious about this, but is a workflowrun materials really completeness true (from `NewWorkflowRun`)?

For generic generator, I'm not sure

Reply:
Completeness gets taken from the `WorkflowRun`. The generic generator only supplies `completeness.parameters = true` based on a suggestion from @MarkLodato. All the rest of the fields are false.

https://github.com/slsa-framework/slsa-github-generator/blob/main/slsa/github.go#L117-L119

See https://github.com/slsa-framework/slsa-github-generator/pull/3#discussion_r840747519 for context when it was added.

Reply:
I'm going to resolve as I think there isn't anything to do here but feel free to reopen if necessary.